packages/dev/s2-docs/pages/s2/Picker.mdx

```diff
@@ -83,6 +83,29 @@ import UserSettings from '@react-spectrum/s2/icons/UserSettings';
 </Picker>
 ```
 
+```tsx render
+"use client";
+import {Avatar, Picker, PickerItem, Text} from '@react-spectrum/s2';
+
+const users = Array.from({length: 10}, (_, i) => ({
+  name: `User ${i + 1}`,
+  email: `user${i + 1}@example.com`,
+  avatar: `https://i.imgur.com/kJOwAdv.png`
+}));
+
+<Picker label="Share" items={users}>
+  {(item) => (
+    <PickerItem textValue={item.name}>
+      {/*- begin highlight -*/}
+      <Avatar slot="avatar" src={item.avatar} />
+      {/*- end highlight -*/}
+      <Text slot="label">{item.name}</Text>
+      <Text slot="description">{item.email}</Text>
+    </PickerItem>
+  )}
+</Picker>
+```
```

Comment:
Not sure if this doc example is good enough for the site, or if it's okay to have two `tsx render` blocks one after another like I did here. Same thoughts for the ComboBox docs Iadded